### PR TITLE
Reintroduce missing custom metrics and metadata to device page.

### PR DESCRIPTION
**Pull request by elinol**
Also stop saving firmware metadata on device health.

### CODE REVIEW

**Comment by elinol** [comment]
image

**Comment by joshk** [comment]
Thanks for including the screenshot. 

I would recommend not showing custom metrics in the health section on the Device show page as it will mess with formatting. 

Can we keep custom metrics on the Device Metrics page only?

**Comment by lawik** [comment]
As clarified on Slack, this restores functionality that was broken with metrics-changes. It needs to be nicer in the next iteration of the UI. But it does matter.

**Review by joshk — APPROVED** [review]
I approve this under duress 😆

**Comment by lawik** [comment]
I would prefer if we remove the "stop saving firmware metadata" bit from this part for now. That's mixing two different topics. That merge is intentional and at some point I can explain more of my thinking there. It may turn out to be unnecessary but I think it will be helpful to have.

File: lib/nerves_hub_web/channels/device_channel.ex [hunk 1]
```diff
@@ -409,18 +409,12 @@ defmodule NervesHubWeb.DeviceChannel do
...
-          into: %{},
-          do: {to_string(key), to_string(val)}
-
     # Separate metrics from health report to store in metrics table
     metrics = device_status["metrics"]
 
     health_report =
       device_status
       |> Map.delete("metrics")
-      |> Map.put("metadata", Map.merge(device_status["metadata"], device_meta))
```
Comment (lawik) [inline lib/nerves_hub_web/channels/device_channel.ex:423]: I'd rather this not be in this PR :)

**Review by lawik — CHANGES_REQUESTED** [review]
Small thing about that merge bit.

Thanks for fixing the thing :)

**Review by lawik — APPROVED** [review]

**Comment by lawik** [comment]
Thank you very much ✨

**Pull request merged**